types/sync-storage/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "dtslint/dt.json", "extends": "@definitelytyped/dtslint/dt.json" }
```

Comment:
You only need the first or the second key, not both. The readme currently prescribes `{ "extends": "@definitelytyped/dtslint/dt.json" }`, so I'd go with that.